package.json

```diff
@@ -62,7 +62,7 @@
     "polymer-analyzer": "2.0.0-alpha.18",
     "polymer-build": "^0.5.1",
     "polymer-project-config": "^1.0.0",
-    "polyserve": "0.16.0-prerelease.3",
+    "polyserve": "^0.16.0-prerelease.4",
```

Comment:
I think we want to pin exactly for now, would could have some breaking changes before release.